config/clients/python/template/README_calling_api.mustache

```diff
@@ -117,6 +117,9 @@ async def write_authorization_model():
     api_instance = open_fga_api.OpenFgaApi(api_client)
```

Comment:
can we use variables in the config for these? (same for python tests)

Reply:
Could be a separate PR though

Reply:
let's do it in separate PR.